required/tools/array.dtx

```diff
@@ -2144,10 +2327,61 @@
 %    start the wanted =\halign=.
 %    \changes{1994/12/08}{v2.3b}{add \cs{tabularnewline}}
 %    \begin{macrocode}
-  \let\\\@arraycr \let\tabularnewline\\\let\par\@empty \@preamble}
+  \let\\\@arraycr \let\tabularnewline\\\let\par\@empty
```

Comment:
around 10 lines above (where I can't comment ...) is \m@th used. In the current version of the math-code it disables math grabbing which we don't want here (as it disables math inside all cells), so that should be changed into `\mathsurround\z@`.

Reply:
where you can't comment?  why not? you should be able to see more of the surrounding by clicking on one of the arrows and then you shouldd be able to comment (or amy I mssing something?). But otherwise point taken :-)

Reply:
well I can't. Comments can be added around 3 or four lines around a changed line but not in arbitrary places.

Reply:
actually, if the latex-lab-math code is correctly implemented then \m@th should only trigger a signal that we have fake math here and that shouldn't stop grab inner math only the math at this level. Right now that's not the case, but it should be in the end (as \m@th is only a heuristic).

Reply:
Well we do *not* have fake math here and this occurence of `\m@th` should not trigger a signal. So we should use here `\mathsurround` or remove the redefinition of `\m@th` from the math code (or do both). (I don't think that removing the redefinition will lead to problems, url is still correctly tagged. ).  Which one do you prefer.

Reply:
> well I can't. Comments can be added around 3 or four lines around a changed line but not in arbitrary places.

Support for this feature is on GitHub's public roadmap, but in category "Future".
https://github.com/github/roadmap/issues/281

Also I find in GitHub Rest API to [Create a review comment for a pull request](https://docs.github.com/en/rest/pulls/comments?apiVersion=2022-11-28#create-a-review-comment-for-a-pull-request), the value of `line` parameter is limited to

> The line of the blob in the pull request diff that the comment applies to. For a multi-line comment, the last line of the range that your comment applies to.